samples/Skript/command-hello.sk

```diff
@@ -0,0 +1,6 @@
+# GitHub Lingust Example Skript
+
+on command /hello:
+  permission: hi.hello
+  trigger:
+    send "Hello!" to player
```

Comment:
The samples should be more realistic/representative of the language. For instance these are very basic and are missing some of the syntax shown on the website like {} and %%.

Reply:
I'll fix these in the morning.